aqua/LyX/Portfile

```diff
@@ -7,6 +7,7 @@ PortGroup           cxx11 1.1
 name                LyX
 conflicts           LyX1
 version             2.3.2
+revision            1
```

Comment:
One may argue that this isn't necessary because `port rev-upgrade` will rebuild the port for affected users anyway. Any opinions?

Reply:
Always best to avoid relying on rev-upgrade, so i would leave it.